agenta-cli/agenta/cli/helper.py

```diff
@@ -1,11 +1,35 @@
-from pathlib import Path
-from typing import Any, List, MutableMapping
-import click
+import sys
 import toml
+import click
+import questionary
+from pathlib import Path
 from agenta.client import client
+from typing import Any, List, MutableMapping
 from agenta.client.api_models import AppVariant
 
 
+def get_api_key():
+    agenta_dir = Path.home() / ".agenta"
+    credentials_file = agenta_dir / "credentials"
+
+    if credentials_file.is_file():
+        with credentials_file.open("r") as f:
+            for line in f:
+                key, value = line.strip().split("=")
+                if key == "api_key":
+                    return value
+    else:
+        api_key = questionary.text("Please provide your API key:").ask()
```

Comment:
Let's add here a link where the user can get it: 
https://demo.agenta.ai/settings?tab=apiKeys